Fixes
semaphore.c:180:5: missing terminating " character
semaphore.c:213:9: missing terminating " character
make[1]: *** [semaphore.o] Error 1
make[1]: Leaving directory `/home/dank/queue/jobdir.fast/crosstool-dev/build/arm-unknown-linux-gnu/gcc-3.3.4-glibc-2.2.5/linux-2.4.26/arch/arm/kernel'
make: *** [_dir_arch/arm/kernel] Error 2

when building kernel.org 2.4.26 for arm with gcc-3.3.x
(Note: one probably wants to apply ftp://ftp.arm.linux.org.uk/pub/armlinux/source/kernel-patches/v2.4/patch-2.4.26-vrs1.bz2
instead of my patches when really building 2.4.26 for arm;
my patches only include the bare minimum to get the vanilla kernel to compile.)

--- linux-2.4.26/arch/arm/kernel/semaphore.c.old	Mon Mar 21 06:50:38 2005
+++ linux-2.4.26/arch/arm/kernel/semaphore.c	Mon Mar 21 06:50:55 2005
@@ -193,7 +193,7 @@
 	bl	__down_interruptible		\n\
 	mov	ip, r0				\n\
 	ldmfd	sp!, {r0 - r3, pc}^		\n\
-
+						\n\
 	.align	5				\n\
 	.globl	__down_trylock_failed		\n\
 __down_trylock_failed:				\n\
